src/indra_cogex/sources/nih_reporter/__init__.py

```diff
@@ -241,6 +259,17 @@ def download_files(
             )
 
 
+def clean_text(text: Any) -> Any:
```

Comment:
this function must be rewritten several places throughout the project if you want to try and wrangle them

Reply:
This is the only place so far (to my knowledge) where this is needed, so maybe we can keep it as is for now?

Reply:
Yes fine for now :)